### PR TITLE
LPS-101575 Deprecate unused js utilities in frontend-aui-web

### PR DESCRIPTION
**Pull request by mateomustapic**
The goal of this task is to deprecate unused JS utilities in **frontend-aui-web**.
Currently the only function which needs to be deprecated is **toNumber**.

### CODE REVIEW

**Comment by mateomustapic** [comment]
ci:test:sf

**Comment by mateomustapic** [comment]
ci:test:stable

**Comment by mateomustapic** [comment]
ci:test:relevant

**Comment by mateomustapic** [comment]
ci:stop

**Comment by mateomustapic** [comment]
ci:test:sf

**Comment by mateomustapic** [comment]
ci:test:stable

**Comment by mateomustapic** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:stop

**Comment by julien** [comment]
ci:test:relevant

**Comment by mateomustapic** [comment]
Hi @john-co can you please take a look at this CI error, it shouldn't be related with this minor change in the PR.

Thanks in advance!

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js [hunk 1]
```diff
@@ -920,6 +920,9 @@
 			form.submit();
 		},
 
+		/**
+		 * @deprecated As of Athanasius (7.3.x), with no direct replacement
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js:924]: This should probably be: "replaced by `parseInt()`".

In other places like this we have pointed to the built-in JS functionality or DOM API  (see examples like [this one](https://github.com/liferay/liferay-portal/blob/3efd054efcdb2625a857906ca3a2a5a718941897/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L575) or [this one](https://github.com/liferay/liferay-portal/blob/3efd054efcdb2625a857906ca3a2a5a718941897/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L582)).

**Comment by john-co** [comment]
I think this is unrelated to the PR, requesting CI team to take a look:
```
     [exec] Error response from daemon: endpoint cloud-10-0-13-166.lax.liferay.com_mysql not found
```

**Comment by julien** [comment]
@john-co thanks for that, I also realized we'll have to update our PR according to what @wincent suggested [above](https://github.com/julien/liferay-portal/pull/274#discussion_r418040770).

**Comment by john-co** [comment]
Confirmed with CI team. Most likely a network issue and is unrelated.

**Pull request closed without merging**